tests/test_order.py

```diff
@@ -3,7 +3,7 @@
 import pytest
 
 from up42.asset import Asset
-from up42.order import Order
+from up42.order import Order, _translate_construct_parameters
```

Comment:
We don't test the hidden functions. Please parameterize the tests of order placement and estimation for the case of tasking